plugins/texteditor.koplugin/main.lua

```diff
@@ -167,23 +170,32 @@ Export text to QR code, that can be scanned, for example, by a phone.]]),
                         self.qr_code_export = not self.qr_code_export
                     end,
                 },
+                {
+                    text = _("Show keyboard on start"),
+                    checked_func = function()
+                        return self.show_keyboard_on_start
```

Comment:
This setting is more worthy / "core" text editor than "QR code export", so may be move it above "QR code export" ?
Also, a separator between the settings related to paragraph direction and this Keyboard+QR could be nice ?